typedspark/_schema/get_schema_definition.py

```diff
@@ -35,16 +35,53 @@ def get_schema_definition_as_string(
     return imports + schema_string
 
 
-def _get_comment(schema: Type[Schema], col_name: str) -> str:
-    """Return the comment of a given column."""
-    if (
-        hasattr(schema.__annotations__[col_name], "__metadata__")
-        and schema.__annotations__[col_name].__metadata__ is not None
-    ):
-        comment = schema.__annotations__[col_name].__metadata__[0]
+def _create_docstring(schema: Type[Schema]) -> str:
+    """Create the docstring for a given ``Schema``."""
+    if schema.get_docstring() is not None:
+        docstring = f'    """{schema.get_docstring()}"""\n\n'
     else:
-        comment = ""
-    return comment
+        docstring = '    """Add documentation here."""\n\n'
+    return docstring
+
+
+def _extract_comment(typehint: str) -> tuple[str, str]:
+    """Extract the comment from a typehint."""
+    comment = ""
+    if "Annotated" in typehint:
+        typehint, comment = re.search(r"Annotated\[(.*), '(.*)'\]", typehint).groups()
+    return typehint, comment
+
+
+def _create_typehint_comment(col_type) -> list[str]:
```

Comment:
Can you change the type to `typle[str, str]`? To make it explicit that there are two entries?

Reply:
done